bincomp/Program.cs

```diff
@@ -117,6 +115,16 @@ private static void PrintUsage()
 			Console.WriteLine("bincomp.exe [input.asm] [output.json] [symbols.txt]");
 		}
 
+		private static void CmdReadOutput(object sender, DataReceivedEventArgs e, AutoResetEvent outputWaitHandle, StringBuilder output)
+		{
+			if (e.Data == null)
+			{
+				outputWaitHandle.Set();
+				return;
+			}
+			output.AppendLine(e.Data);
```

Comment:
Asking out of ignorance/curiousity. Does the `AutoResetEvent` need to be reset at the end of the function (if `e.Data != null`)?

Reply:
No actual need to Reset() the event.  It was started up in the Reset() state already.  Once there is no more data, it gets Set(), which allows the process to continue.